russian/light-fire/so-scary/so-scary.md

```diff
@@ -1,3 +1,5 @@
 Вы остаетесь в комнате. Расплавленный зефир потихоньку затапливает помещение.
 И вот горячая сладкая жидкость уже достигла ваших колен. Но вы не отчаиваетесь
-и успеваете съедать зефир по мере его поступления.
\ No newline at end of file
+и успеваете съедать зефир по мере его поступления.
+
+[Вы просыпаетесь.](awaken/awaken.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.